template_example/dev/ansible/00_create_intention.yml

```
@@ -0,0 +1,16 @@
+- name: set variable
+  set_fact:
+    consul_token: "master"
```

Could you change it to a dynamic lookup?

```suggestion
    consul_token: "{{ lookup('env', 'consul_master_token') }}"
```